test/DADBridgeWrapper.ts

```diff
@@ -0,0 +1,163 @@
+import Contracts from '../components/Contracts';
+import { TestERC20Token } from '../typechain';
+import { DADBridgeWrapper } from '../typechain';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { expect } from 'chai';
+import { utils, BigNumber } from 'ethers';
+import { ethers } from 'hardhat';
+
+const ROLE_ADMIN = utils.id('ROLE_ADMIN');
+const TOTAL_SUPPLY = BigNumber.from(1_000_000_000);
+const MINT_AMOUNT = BigNumber.from(1_000_000);
+const INVALID_TOKEN = '0x'.padEnd(42, '0');
```

Comment:
Let's just use `AddressZero` here. There's no need to redefine it explicitly

Reply:
Replaced with:
```
const INVALID_TOKEN = ethers.constants.AddressZero;
const INVALID_RECIPIENT = ethers.constants.AddressZero;
const INVALID_AMOUNT = ethers.constants.Zero;
```